citadel-internal-service-types/src/lib.rs

```diff
@@ -570,6 +573,11 @@ pub struct FileTransferTickNotification {
     pub cid: u64,
     pub peer_cid: Option<u64>,
     pub status: ObjectTransferStatus,
+    pub request_id: Option<Uuid>,
```

Comment:
Potentially the same as with the other struct. Take a look at the clippy workflow to see the test it shows an error for.